### ISSUE TITLE
tonic support

### ISSUE THREAD

**Issue by lcmgh** [Ptrskay3/axum-prometheus#67]
I'd love this to use with `tonic`

**Comment by Ptrskay3** [comment]
I agree. I don't have the capacity right now to implement this, but if anyone's interested to work on this, then just ping me. I'll be able to provide some extra info or pointers.
